lms/djangoapps/lms_xblock/mixin.py

```diff
@@ -15,6 +15,9 @@
 # more information can be found here: https://openedx.atlassian.net/browse/PLAT-902
 _ = lambda text: text
 
+INVALID_USER_PARTITION_VALIDATION = _(u"This component refers to deleted or invalid user partitions.")
```

Comment:
@catong The only way to get into this situation is to refer to a UserPartition (like Cohort, Random, or EnrollmentTrack) that is then removed from the system. So basically it's very, very unlikely someone would ever see it. I can use the suggested wording-- at this point, I think we are keeping "Group Configurations" as the name of the setting tab in Studio, even when it also contains Enrollment Tracks.